zen/dex/bench.py

```
@@ -1,361 +1,361 @@
+            self.log(f"{attacker.name} executes {hits}x speed blitz!")
+            
+        elif move == BattleMove.DEFEND:
+            # Defensive move, small counter damage
+            base_damage = attacker.defense * 0.3
+            attacker.defense *= 1.5  # Temporary defense boost
+            self.log(f"{attacker.name} takes defensive stance!")
+            
+        elif move == BattleMove.SPECIAL:
+            # Special ability attack
+            if attacker.abilities:
+                ability = random.choice(attacker.abilities)
```

Ruff (0.16.0)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)